projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order/header.ts

```diff
@@ -34,3 +34,42 @@ export function headerTabbingOrder(
 
   verifyTabbingOrder(containerSelector, config);
 }
+
+export function subCategoryTabbingOrder(
+  config: TabElement[],
+  subCategoryName: string,
+  mobile: boolean = false
+) {
+  cy.visit('/');
+
+  if (mobile) {
+    cy.viewport(formats.mobile.width, formats.mobile.height);
+    cy.get('cx-hamburger-menu button')
+      .first()
+      .click()
+      .focus();
+  }
+
+  cy.get('cx-category-navigation').within(() => {
+    cy.get('cx-navigation-ui')
+      .find('nav')
+      .should('have.length', 30);
+    cy.get('cx-navigation-ui nav h5')
+      .contains(subCategoryName)
+      .should('be.visible')
+      .focus()
+      .trigger('keydown', {
+        key: ' ',
+        code: 'Space',
+        force: true,
+      });
+  });
+
+  cy.get('cx-navigation-ui nav[class="is-open"] h5')
```

Comment:
Should we focus the first element, since we have the new way of checking tabbing?

Reply:
btw. I would prefer to use CSS selectors - `nav.is-open` instead of the brackets

Reply:
We need to focus the first element to invoke the focus class to make the wrapper visible.